app/src/main/java/com/geekskool/dhobi/DbHelper.java

```diff
@@ -0,0 +1,44 @@
+package com.geekskool.dhobi;
+
+import android.content.Context;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+
+import com.geekskool.dhobi.Contracts.Contract;
+
+/**
+ * Created by manisharana on 17/2/17.
+ */
+public class DbHelper extends SQLiteOpenHelper {
+    private static final int DATABASE_VERSION = 1;
+    private static final String DATABASE_NAME = "vipasanaDatabase.db";
```

Comment:
Keep configuration values in a central location (perhaps a `resource` file, in Java), rather than as fields on types.